gravitee-apim-portal-webui-next/src/components/nav-bar/nav-bar.component.ts

```diff
@@ -35,9 +35,7 @@ export class NavBarComponent {
   @Input()
   siteTitle!: string;
 
-  @Input()
-  customLinks?: PortalMenuLink[];
-
+  customLinks: InputSignal<PortalMenuLink[]> = input([<PortalMenuLink>{}]);
```

Comment:
```suggestion
  customLinks: InputSignal<PortalMenuLink[]> = input([]);
```

Reply:
Actually this will work 🔽 
```suggestion
  customLinks: InputSignal<PortalMenuLink[]> = input<PortalMenuLink[]>([]);
```